EIPS/eip-1193.md

```diff
@@ -366,7 +376,13 @@ All supported RPC methods **MUST** be identified by unique strings.
 
 Providers **MAY** support whatever RPC methods required to fulfill their purpose, standardized or otherwise.
 
-If a Provider supports a method defined in a finalized EIP, the Provider's implementation of this method **MUST** adhere to the method's specification.
+If a Provider supports a method defined in a finalized EIP, the Provider's implementation of this method **MUST** adhere to the method's specification. This includes:
+
+- The method name, given as the string value of `RequestArguments.method`
+- The method parameters, given as the value of `RequestArguments.params`, either:
+  - By position, in an array
+  - By name, in an object
+- Any additional properties on `RequestArguments`, as necessary
```

Comment:
I don't think it is necessary for this to be part of the specification.  The specification only needs to describe the set of things required for the two systems to communicate, it doesn't need to describe all possible things that one may wish to have.  For example, an HTTP specification may state that certain headers are required, but it needn't specify that all other headers are _allowed_ (though not relevant to the specification).